server/start.sh

```diff
@@ -2,6 +2,15 @@
 
 echo "Initializing Immich $IMMICH_SOURCE_REF"
 
+cat <<EOF > /usr/src/app/upload/upload/README.txt
```

Comment:
```suggestion
cat << 'EOF' > /usr/src/app/upload/upload/README.txt
```

I don't think there is a need for variable interpolation in this block.

Is there a way we can refer to the `WORKDIR` instead of hard coding the /usr/src... patch that will probably change in the future?

Reply:
I don't have enough knowledge to make this happen, maybe one of the developers can help with this.

Reply:
Tbh I would just not do this. I'd just add a comment in the compose (and maybe .env?) file.